lib/ibandit/iban.rb

```diff
@@ -512,5 +512,12 @@ def pseudo_iban?(input)
       return false if input.nil?
       input.slice(2, 2) == Constants::PSEUDO_IBAN_CHECK_DIGITS
     end
+
+    def account_number_not_all_zeros?
+      return true unless @swift_account_number.to_s.chars.uniq == ["0"]
```

Comment:
i prefer this to be written as: `return true if @swift_account_number.to_s.chars.uniq != ["0"]`

Reply:
Should we just lose the negation and handle the not in the check above? E.g.:
```ruby
def account_number_all_zeros?
  return @swift_account_number.to_s.chars.uniq == ["0"]
end
````

Reply:
@vinayvinay Rubocop doesn't allow that (in the rest of our codebases) so I've stopped writing it that way. However, here, we're good! So I've changed it 😬 

@ivgiuliani Doesn't fit the pattern that we use here where we rely on the private methods inserting errors into the `@errors` array.